spec/discourse_api/api/users_spec.rb

```diff
@@ -284,4 +284,23 @@
       expect(result.status).to eq(200)
     end
   end
+
+  describe "#check_username" do
+    let(:url) { "http://localhost:3000/users/check_username.json?username=sparrow&api_key=test_d7fd0429940&api_username=test_user" }
+
+    before do
+      body = '{"available":false,"suggestion":"sparrow1"}'
+      stub_get(url).to_return(body: body, headers: { content_type: "application/json" })
+    end
+
+    it "makes the correct delete request" do
```

Comment:
Looks like you need to update the text here.

Reply:
Oof! Good catch.

Reply:
@oblakeerickson fixed